ui/shared/components/panel/family/FamilyFields.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Popup } from 'semantic-ui-react'
+import styled from 'styled-components'
+
+import { loadAnalystOptions } from 'redux/rootReducer'
+import {
+  getSamplesByFamily,
+  getUserOptionsIsLoading,
+  getHasActiveVariantSampleByFamily,
+} from 'redux/selectors'
+
+import Sample from '../sample'
+import { ColoredIcon } from '../../StyledComponents'
+import { Select } from '../../form/Inputs'
+import DataLoader from '../../DataLoader'
+import { getAnalystOptions } from '../../../../pages/Project/selectors'
+
+const NoWrap = styled.div`
+  white-space: nowrap;
+`
+
+const BaseFirstSample = React.memo(({ firstFamilySample, compact, hasActiveVariantSample }) =>
+  <Sample
+    loadedSample={firstFamilySample}
+    hoverDetails={compact ? 'first loaded' : null}
+    isOutdated={!hasActiveVariantSample}
+  />,
+)
+
+BaseFirstSample.propTypes = {
+  firstFamilySample: PropTypes.object,
+  compact: PropTypes.bool,
+  hasActiveVariantSample: PropTypes.bool,
+}
+
+const mapSampleDispatchToProps = (state, ownProps) => ({
+  firstFamilySample: (getSamplesByFamily(state)[ownProps.familyGuid] || [])[0],
+  hasActiveVariantSample: getHasActiveVariantSampleByFamily(state)[ownProps.familyGuid],
+})
+
+export const FirstSample = connect(mapSampleDispatchToProps)(BaseFirstSample)
+
+const BaseAnalystEmailDropdown = React.memo(({ load, loading, onChange, value, ...props }) =>
+  <DataLoader load={load} loading={false} content>
+    <Select
+      loading={loading}
+      additionLabel="Assigned Analyst: "
+      onChange={val => onChange(val)}
+      value={value}
+      placeholder="Unassigned"
+      search
+      {...props}
+    />
+  </DataLoader>,
+)
+
+BaseAnalystEmailDropdown.propTypes = {
+  load: PropTypes.func,
+  loading: PropTypes.bool,
+  onChange: PropTypes.func,
+  value: PropTypes.any,
+}
+
+const mapDropdownStateToProps = state => ({
+  loading: getUserOptionsIsLoading(state),
+  options: getAnalystOptions(state),
+})
+
+const mapDropdownDispatchToProps = {
+  load: loadAnalystOptions,
+}
+
+BaseAnalystEmailDropdown.propTypes = {
+  load: PropTypes.func,
+  loading: PropTypes.bool,
+  onChange: PropTypes.func,
+  value: PropTypes.any,
+}
```

Comment:
Lines #59-64 are identical to lines #75-80. It seems an issue existing in the previous file.